### ISSUE TITLE
Bypass First Authentication Page

### ISSUE THREAD

**Issue by mrdenny** [psignoret/aad-sso-wordpress#108]
When logging in via this plugin it first takes you to the generic Microsoft Authentication page. Is there a way to have it take us directly to the co-branded version of the page bypassing the generic page and bypassing having it ask if you should use your AAD account or you Microsoft Account (Life ID)?

**Comment by psignoret** [comment]
Yes, the "Domain hint" setting will allow you to indicate to Azure AD which domain name you will be using, which will be used to load custom branded details (and if your domain name is federated, will automatically trigger the redirect to your on-premises identity provider):

![image](https://cloud.githubusercontent.com/assets/231140/21906614/3f74c630-d90c-11e6-895e-2b912f7ae6e9.png)

However, if you have both a consumer Microsoft Account (MSA, formerly known as LiveID) and an Azure AD account with the same username, then you might still be prompted to disambiguate. The only thing you can do here (and really, you _should_ do this anyway) is to remove the ambiguity by renaming your personal account.

 * Blog post discussing the overlap: https://blogs.technet.microsoft.com/enterprisemobility/2016/09/15/cleaning-up-the-azure-ad-and-microsoft-account-overlap/
 * Step by step instructions for renaming your MSA: https://support.microsoft.com/en-us/help/11545/microsoft-account-rename-your-personal-account
